docker-compose-base.yml

```diff
@@ -264,21 +267,22 @@ services:
     stdin_open: true
     tty: true
     environment:
+      # Uncomment these lines to enable single sign on if you use CAS
+      # CAS_SERVER_URL: https://your.cas.server
+      # CSRF_COOKIE_DOMAIN: .aether.local
+
+      DEBUG: "true"
+      DJANGO_SECRET_KEY: ${UI_DJANGO_SECRET_KEY}
+      DJANGO_STORAGE_BACKEND: filesystem
```

Comment:
I guess this can be taken out now that you removed `django-storages` from UI

Reply:
Nop, it's mandatory to indicate it even if you are not using it:

https://github.com/eHealthAfrica/aether/blob/release-0.9/aether-common-module/aether/common/conf/settings.py#L249